File: Nicollas/integracao.py
from flask import Flask, jsonify

app = Flask(__name__)


@app.route("/estatisticas")
def estatisticas():
    dados = {
        "Matemática": 10,
        "História": 8,
        "Biologia": 6,
        "Português": 7
    }
    return jsonify(dados)


if __name__ == "__main__":
    app.run(debug=True)

File: Nicollas/integracao.js
const botao = document.getElementById("carregar");
const resultado = document.getElementById("resultado");

botao.addEventListener("click", function() {
    const materiais = [
        { disciplina: "Matemática", quantidade: 10 }, 
        { disciplina: "História", quantidade: 8 }, 
        { disciplina: "Biologia", quantidade: 6 }
    ];

    resultado.innerHTML = "";